raft/raft_paper_test.go

```diff
@@ -617,30 +617,31 @@ func TestFollowerCheckMessageType_MsgAppend2AB(t *testing.T) {
 func TestFollowerAppendEntries2AB(t *testing.T) {
 	tests := []struct {
 		index, term uint64
+		lterm       uint64
 		ents        []*pb.Entry
 		wents       []*pb.Entry
 		wunstable   []*pb.Entry
 	}{
 		{
-			2, 2,
+			2, 2, 3,
 			[]*pb.Entry{{Term: 3, Index: 3}},
 			[]*pb.Entry{{Term: 1, Index: 1}, {Term: 2, Index: 2}, {Term: 3, Index: 3}},
 			[]*pb.Entry{{Term: 3, Index: 3}},
 		},
 		{
-			1, 1,
+			1, 1, 4,
 			[]*pb.Entry{{Term: 3, Index: 2}, {Term: 4, Index: 3}},
 			[]*pb.Entry{{Term: 1, Index: 1}, {Term: 3, Index: 2}, {Term: 4, Index: 3}},
 			[]*pb.Entry{{Term: 3, Index: 2}, {Term: 4, Index: 3}},
 		},
 		{
-			0, 0,
+			0, 0, 1,
```

Comment:
```suggestion
			0, 0, 2,
```
This test is about how the implement deal with existing logs and the logs in `MsgAppend`, set the `lterm` to 1 may cause the message reject by term check before handle.

Reply:
Yes, leader's term should be greater than/equals to follower, otherwise will be rejected. Thank you for your suggestion.